README.md

```diff
@@ -5,16 +5,17 @@
 
 ## rust-lean-models
 
-Lean models of various Rust libraries to facilitate Lean-based verification of Rust programs.
+Lean models of various functions from Rust libraries to facilitate Lean-based verification of Rust programs.
```

Comment:
I think the models extend beyond just functions since we model a Rust `String` as a list of characters in Lean for example.

Perhaps we should say data structures or data types instead?

Reply:
Yes. that makes sense.